CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1.5.0 (08/04/2020)
+
+**Common**
+
+* `DefaultCache` implementation moved to pimpl.
+* Various warnings fixed.
+* CMake now uses boost repository instead of downloading and unpacking the archive.
+* Method `OlpClientSettingsFactory::CreateDefaultCache` now returns nullptr in case if failed to open a disk cache.
```

Comment:
```suggestion
* Method `OlpClientSettingsFactory::CreateDefaultCache` now returns `nullptr` in case it failed to open one of the user-defined disk caches.
```